traces-backend-newrelic.gemspec

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative 'lib/traces/backend/newrelic/version'
+
+Gem::Specification.new do |s|
+  s.name = 'traces-backend-newrelic'
+  s.version = Traces::Backend::NewRelic::VERSION
+  s.required_ruby_version = '>= 2.4.0'
+  s.authors = ['Tanna McClure', 'Kayla Reopelle', 'James Bunch', 'Hannah Ramadan']
+  s.licenses = ['Apache-2.0']
+  s.description = 'New Relic implementation for the Traces gem'
+  s.email = 'support@newrelic.com'
+  s.homepage = 'https://github.com/newrelic/traces-backend-newrelic'
+  s.summary = 'A traces backend for New Relic'
+
+  s.files = Dir.glob('{lib}/**/*', File::FNM_DOTMATCH, base: __dir__)
+
+  s.add_dependency 'newrelic_rpm'
```

Comment:
Is there any reason we'd want to set a minimum agent version here? Like anything greater than 9.0?

Reply:
hmm i'm not sure. I don't really think that's necessary bc the agent methods used have been around for a long time. But if you have any reasons in mind i might not have thought of just lmk!

Reply:
Good point! I don't think I realized we were using such old methods. I can't think of any other reasons!